src/settings.moon

```diff
@@ -70,7 +81,7 @@ settings = {
 	-- [[ chapter marker color options ]] --
 	-- color of chapter marker before/after it has been passed. BGR hex.
 	'chapter-marker-before': '7A77F2'
-	'chapter-marker-after': '2D2D2D'
+	'chapter-marker-after': BG_PLACEHOLDER
```

Comment:
You seem to have accidentally picked this up in find/replace. This doesn't match the `-background` suffix check, so I don't think it'll work properly.